### PR TITLE
Document fixedBaseFee genesis option

### LINKED ISSUE #1550: Document `fixedBaseFee` genesis option
Document `fixedBaseFee` genesis file option. [This page](https://besu.hyperledger.org/development/public-networks/reference/genesis-items#configuration-items) might need to be updated, and maybe some other locations if applicable – look into https://github.com/hyperledger/besu/pull/6562 for details.

### PR DESCRIPTION
**Pull request by joaniefromtheblock**
Add `fixedbasefee` option to docs. 

Fixes #1550

### CODE REVIEW

**Review by macfarla — APPROVED** [review]

File: docs/public-networks/reference/genesis-items.md [hunk 1]
```diff
@@ -29,6 +29,7 @@ Network configuration items are specified in the genesis file in the `config` ob
 | `ecCurve`           | Specifies [the elliptic curve to use](../../private-networks/how-to/configure/curves.md). Default is `secp256k1`.                                                                                                 |
 | `discovery`         | Specifies [discovery configuration items](#discovery-configuration-items). The `discovery` object can be left empty.                                                                                              |
 | `zeroBaseFee`       | Specifies a base fee of `0` for [free gas networks](../../private-networks/how-to/configure/free-gas.md#4-enable-zero-base-fee-if-using-london-fork-or-later).                                                      |
+| `fixedBaseFee`      | Specifies a constant base fee for blocks, overriding the dynamic base fee calculation of [EIP-1559].  |
```
Comment (bgravenorst) [inline docs/public-networks/reference/genesis-items.md:32]: This appears to be missing a link.

**Review by bgravenorst — COMMENTED** [review]
Left a comment.

File: docs/public-networks/reference/genesis-items.md (hunk 1 above)
Comment (joaniefromtheblock) [reply docs/public-networks/reference/genesis-items.md]: Thanks @bgravenorst . Please review

**Review by bgravenorst — APPROVED** [review]
lgtm

**Pull request merged**